### DISCUSSION TITLE
Manage Data

### DISCUSSION THREAD

**Discussion by alebedev777** [milvus-io/milvus#26651] [Q&A and General discussion] [answered]
Milvus version : milvusdb/milvus:v2.3.0-gpu-beta
Deployment mode : standalone
OS : Ubuntu
when milvus run with 2 collections:
![server capabilities](https://github.com/milvus-io/milvus/assets/55959178/27fc7623-daca-45bc-bdc3-25a2c8bdb12e)

I create 5 collections with dim 2048 ![Screenshot from 2023-08-28 17-23-12](https://github.com/milvus-io/milvus/assets/55959178/332f5a4a-f67e-49b4-ba66-71a9de077a64)

and 3 collections with dim 768
![Screenshot from 2023-08-28 17-44-31](https://github.com/milvus-io/milvus/assets/55959178/25690a92-83ee-4869-8dc0-193eaf8bd67d) 

for collections i create index as
`INDEX_PARAMS = {
  "metric_type":"L2",
  "index_type":"IVF_PQ",
  "params":{"nlist": 5000, "m": dim/2},
}`

when I load the data in two collections (768 | 2048) and data count in collections becomes more than 1M (for example collection dim 768 - 680K | dim 2048 - 380K) milvus crashes and freezes. throws error `pymilvus.exceptions.MilvusException: <MilvusException: (code=1, message=proxy not healthy)>`. after rebuld milvus one of collections don't load and i can't work with collection.

Could you please guide me where am I making a mistake or doing something wrong (index/version/some magic). each collection will have at least 1.5M of data and I would like the collections to be loaded on the server for fast searching (without waiting for the collection to load).

**Comment by xiaofan-luan** [comment]
Hi @alebedev777 
could you try Milvus 2.3?  fixed many bugs in Milvus 2.3.0-beta

**Reply by alebedev777** [reply]
okay. I'll try
i have one more question. i load data (images emb dim 2048 dim and objects emb dim 768) in loop for parts (every 50). Is it correct to load data into collections at the same time or fill one collection first and then start filling the next one?

**Reply by yhmo** [reply]
"fill one collection first and then start filling the next one" is better.

**Comment by alebedev777** [comment]
after changing milvus version from `milvusdb/milvus:v2.3.0-gpu-beta` to `milvusdb/milvus:v2.3.0` search don't work at all. 
first get 
`[search] retry:4, cost: 0.27s, reason: <_InactiveRpcError: StatusCode.UNAVAILABLE, failed to connect to all addresses>
[search] retry:5, cost: 0.81s, reason: <_InactiveRpcError: StatusCode.UNAVAILABLE, failed to connect to all addresses>
[search] retry:6, cost: 2.43s, reason: <_InactiveRpcError: StatusCode.UNAVAILABLE, failed to connect to all addresses>
[search] retry:7, cost: 7.29s, reason: <_InactiveRpcError: StatusCode.UNAVAILABLE, failed to connect to all addresses>
[search] retry:8, cost: 21.87s, reason: <_InactiveRpcError: StatusCode.UNAVAILABLE, failed to connect to all addresses>
[search] retry:9, cost: 60.00s, reason: <_InactiveRpcError: StatusCode.UNAVAILABLE, failed to connect to all addresses>
[search] retry:10, cost: 60.00s, reason: <_InactiveRpcError: StatusCode.UNAVAILABLE, failed to connect to all addresses>
` 
then crashes
`    raise MilvusUnavailableException(message=f"server Unavailable: {timeout_msg}") from e
pymilvus.exceptions.MilvusUnavailableException: <MilvusUnavailableException: (code=1, message=server Unavailable: Retry run out of 10 retry times)>
`
should i refill the collections after changing milvus versions?

**Reply by yhmo** [reply, marked as answer]
v2.3.0 is not compatible with v2.3.0-gpu-beta, I think you need to deploy a clean v2.3.0 instance and recreate the collection.

**Reply by xiaofan-luan** [reply]
why this is not compatible? @yhmo

**Reply by xiaofan-luan** [reply]
I don't know any reason why it should be incompatible

**Reply by yhmo** [reply]
Just ask the related engineer, v2.3.0 is compatible with v2.3.0-gpu-beta. So, I am wrong, this is not a compatible issue. 
@alebedev777 To find out the root cause, create a new issue in the issues panel, put full logs and more description, we will assign it to the proper engineer.
